servers/mlflowserver/mlflowserver/requirements.txt

```diff
@@ -1,6 +1,6 @@
 pyyaml==5.1.2
 requests==2.22.0
-mlflow==1.4.0
+mlflow>=1.4.0
```

Comment:
Could we put a hard limit on the current version? As in `mlflow >= 1.4.0, < 1.9.0`? In the past we've had issues with packages releasing a breaking change, so we usually try to cap the max version.

Reply:
sure, I added the upper bound to 1.9.0